bin/polkadot-archive/Cargo.toml

```diff
@@ -1,20 +1,22 @@
 [package]
 name = "polkadot-archive"
-version = "0.2.2"
+version = "0.2.3"
 authors = ["Andrew Plaza <aplaza@liquidthink.net>"]
 edition = "2018"
 
 [dependencies]
+polkadot-service = { git = "https://github.com/paritytech/polkadot", branch = "master" }
+frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
looks not to be used, I guess it's fetched from other dependencies but still....

```suggestion
```